### PR TITLE
vaunt: update achievement images and add vaunt documentation

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Updated the 1st_place.png and contributor.png Vaunt achievement images.
Added a README.md to the .vaunt directory describing how users can update their profile to show their badges.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/16324
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# 如何在你的 GitHub 个人主页上添加 TiDB 文档挑战赛徽章
+
+[GitHub 个人主页](https://docs.github.com/zh/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)不仅仅是你的 GitHub 仓库的集合，更是你在编程领域中的专业身份标志。
+
+本指南介绍如何在你的 GitHub 个人主页上添加 [2024 TiDB 文档挑战赛](https://asktug.com/t/topic/1019364) (TiDB Docs Dash) 活动徽章。
+
+## 第 1 步：创建一个仓库，存放个人主页 README
+
+如果你已经配置了 GitHub 个人主页 README，请跳过以下步骤，直接进入[第 2 步：编辑你的个人资料 README](#第-2-步编辑个人主页-readme)。
```
Comment (qiancai) [inline .vaunt/add-tidb-docs-dash-badge.md:9]:
```suggestion
如果你已经配置了 GitHub 个人主页 README，请跳过以下步骤，直接进入[第 2 步：编辑个人主页 README](#第-2-步编辑个人主页-readme)。
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# 如何在你的 GitHub 个人主页上添加 TiDB 文档挑战赛徽章
+
+[GitHub 个人主页](https://docs.github.com/zh/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)不仅仅是你的 GitHub 仓库的集合，更是你在编程领域中的专业身份标志。
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:3]: GitHub 官网叫 个人资料，我们也保持统一？
如要更新，其它几处也请替换掉
```suggestion
[GitHub 个人资料页](https://docs.github.com/zh/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)不仅仅是你的 GitHub 仓库的集合，更是你在编程领域中的专业身份标志。
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+1. 在你的 `<github_username>` 仓库的 GitHub 页面上，点击右侧边栏上方的 **Edit README**。
+2. 在 README 文件中添加以下代码，并将 `{{github_username}}` 替换为你的用户名，然后提交更改。
+
+    ```HTML
+    <p>
+      <img src="https://api.vaunt.dev/v1/github/entities/{{github_username}}/achievements?format=svg&limit=3" width="350" />
+    </p>
+    ```
+
+ 此时，刷新你的 GitHub 个人主页，即可看到 TiDB Docs Dash 活动徽章。关于如何管理个人主页 README 的更多信息，请参阅 [GitHub 文档](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)。
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:31]:
```suggestion
 此时，刷新你的 GitHub 个人主页，即可看到 TiDB Docs Dash 活动徽章。关于如何管理个人主页 README 的更多信息，请参阅 [GitHub 文档](https://docs.github.com/zh/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)。
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+2. 以你的 GitHub 用户名命名该仓库。例如，如果你的用户名为 `ilovetidb`，则仓库名必须为 `ilovetidb`。
+3. 选择 **Public** 将该仓库设置为公共可见。
+4. 在 **Initialize this repository with:** 下，选择 **Add a README file**。
+5. 点击 **Create repository**。
+
+## 第 2 步：编辑个人主页 README
+
+1. 在你的 `<github_username>` 仓库的 GitHub 页面上，点击右侧边栏上方的 **Edit README**。
+2. 在 README 文件中添加以下代码，并将 `{{github_username}}` 替换为你的用户名，然后提交更改。
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:23]:
```suggestion
2. 在 README 文件中添加以下代码，并将 `{{github_username}}` 替换为你的 GitHub 用户名，然后提交更改。
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    ```
+
+- 集成你的社区贡献看板到个人主页 README
+
+    社区贡献看板可以通过视图的形式综合展示你在 GitHub 各个仓库的贡献统计数据。
+
+    [![VauntCommunity](https://api.vaunt.dev/v1/github/entities/pingcap/badges/community)](https://community.vaunt.dev/board/pingcap)
+
+    - 查看[你的社区看板](https://community.vaunt.dev/)。
+    - 如需集成你的社区看板，请将以下代码添加到你的 README 文件中，并将 `{{github_username}}` 替换为你的用户名：
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:67]:
```suggestion
    - 如需集成你的社区看板，请将以下代码添加到你的 README 文件中，并将 `{{github_username}}` 替换为你的 GitHub 用户名：
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+    <p>
+        <a href="https://vaunt.dev">
+            <img src="https://api.vaunt.dev/v1/github/entities/{{github_username}}/contributions?format=svg" width="350" />
+        </a>
+    </p>
+    ```
+
+- 集成你的社区贡献看板到个人主页 README
+
+    社区贡献看板可以通过视图的形式综合展示你在 GitHub 各个仓库的贡献统计数据。
```
Comment (hfxsd) [inline .vaunt/add-tidb-docs-dash-badge.md:62]:
```suggestion
    社区贡献看板通过视图的形式，综合展示你在 GitHub 各个仓库的贡献统计数据。
```

**Review by hfxsd — APPROVED** [review]

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+此时，刷新你的 GitHub 个人资料页，即可看到 TiDB Docs Dash 活动徽章。
+
+关于如何管理个人资料 README 的更多信息，请参阅 [GitHub 文档](https://docs.github.com/zh/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme)。
+
+## （可选）通过 Vaunt 添加更多展示
+
+本次 TiDB 文档挑战赛的徽章是通过 [Vaunt](https://vaunt.dev/) 授予的，Vaunt 是一个开发者关系平台，旨在赋能和发展开发者社区。
+
+通过 Vaunt 提供的以下功能，你还可以在你的个人资料 README 中展示你在其他开源项目中做出的贡献：
```
Comment (qiancai) [inline .vaunt/add-tidb-docs-dash-badge.md:39]:
```suggestion
通过 Vaunt 提供的以下功能，你还可以在你的个人资料 README 中展示你在其它开源项目中做出的贡献：
```

File: .vaunt/add-tidb-docs-dash-badge.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+    示例卡片:
+
+    <p>
+        <a href="https://vaunt.dev">
+            <img src="https://api.vaunt.dev/v1/github/entities/jeff1010322/contributions?format=svg" width="350" />
+        </a>
+    </p>
+
+    如需集成你的 developer card，请将以下代码添加到你的 README 文件中，并将 `{{github_username}}` 替换为你的用户名。
+
```
Comment (qiancai) [inline .vaunt/add-tidb-docs-dash-badge.md:52]:
```suggestion
```

**Pull request merged**